build_projects/dotnet-cli-build/CliDependencyVersions.cs

```diff
@@ -7,10 +7,12 @@ namespace Microsoft.DotNet.Cli.Build
 {
     public class CliDependencyVersions
     {
-        public static readonly string SharedFrameworkVersion = "1.0.0-rc3-004459-00";
-        public static readonly string SharedHostVersion = "1.0.1-rc3-004459-00";
+        public static readonly string SharedFrameworkVersion = "1.0.0-rc3-004465-00";
+        public static readonly string SharedHostVersion = "1.0.1-rc3-004465-00";
+        public static readonly string HostFxrVersion = "1.0.1-rc3-004465-00";
```

Comment:
We can't be adding new places with hard-coded versions without updating the 'update-dependencies' script.